.pre-commit-config.yaml

```diff
@@ -1,3 +1,7 @@
+default_install_hook_types:
+  - pre-commit
+  - commit-msg
```

Comment:
Nice, hadn't known about this feature!  Looks like it was added in [2022](https://github.com/pre-commit/pre-commit/blob/main/CHANGELOG.md#2180---2022-04-02).